src/components/input/InputDecorator.tsx

```diff
@@ -46,15 +46,13 @@ const InputDecorator: React.FC<InputDecoratorProps> = ({
 }) => {
   let child;
 
-  if (React.Children.count(children) === 0) {
-    throw new Error('The Input decorator requires one child component.');
-  } else if (React.Children.count(children) > 1) {
+  if (React.Children.count(children) > 1) {
```

Comment:
sweet, does it work well ?

Reply:
It does!
![image](https://user-images.githubusercontent.com/56824367/149174729-1f147f34-42ff-4c2b-85c3-bbfe23fc1d56.png)
![image](https://user-images.githubusercontent.com/56824367/149174828-1180ebd2-c030-4466-a056-cbd73b99689a.png)